.github/workflows/run_shark_tank.yml

```diff
@@ -7,7 +7,7 @@ on:
   # Uncomment `pull_request` below to test your PR on SHARK.
   # Results will be uploaded to gs://shark-benchmark-artifacts.
   # See the workflow's `SHARK -> generate_report` log output for exact results URL.
-  # pull_request:
+  pull_request:
```

Comment:
This workflow has a workflow_dispatch trigger (just above these lines, but GitHub won't let me comment on that...). If you trigger from https://github.com/openxla/iree/actions/workflows/run_shark_tank.yml, you can avoid creating a PR and running all the rest of IREE's CI jobs.

Reply:
Thanks, Scott! This run is on a commit from a week ago. Is there a way to run the workflow on a specific commit without creating a PR?
cc: @mariecwhite

Reply:
Yep: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#providing-inputs
(will require some changes to the workflow file)